File: main_module/src/main/java/com/mobile/main_module/login/view/ILoginView.java
package com.mobile.main_module.login.view;

import com.mobile.common_library.base.IBaseView;
import com.mobile.main_module.login.bean.GetVerificationCode;
import com.mobile.main_module.login.bean.LoginResponse;

public interface ILoginView extends IBaseView {
	
	void getAuthCodeSuccess(GetVerificationCode.DataDTO success);
	
	void getAuthCodeError(String error);
	
	void loginWithAuthCodeSuccess(LoginResponse.DataDTO success);
	
	void loginWithAuthCodeError(String error);
	
}

File: module_main/src/main/java/com/phone/module_main/login/view/ILoginView.kt
package com.phone.module_main.login.view

import com.phone.library_common.base.IBaseView
import com.phone.module_main.login.DataGetVerification
import com.phone.module_main.login.DataLogin

interface ILoginView : IBaseView {

    fun getAuthCodeSuccess(success: DataGetVerification)

    fun getAuthCodeError(error: String)

    fun loginWithAuthCodeSuccess(success: DataLogin)

    fun loginWithAuthCodeError(error: String)

    fun loginSuccess(success: String)

    fun loginError(error: String)

}